app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
+    if len(parts) != 3 or parts[0] != "cp":
+        raise ValueError("Error")
+
+    com, f_name, s_name = parts
```

Comment:
The variable `com` is not used after unpacking. You can omit it or use an underscore `_` to indicate that it's intentionally unused.